FizzBuzz - multiple parameters.rb

```diff
@@ -0,0 +1,58 @@
+def print_fizz_buzz(is_fb)
+  if
+    is_fb%3 == 0 && is_fb%5 == 0
+    puts "FizzBuzz"
+  elsif is_fb%3 == 0
+    puts "Fizz"
+  elsif is_fb%5 == 0
+    puts "Buzz"
+  else
+    puts "No FizzBuzz for you, sport!"
+  end
+end
+
+def fizz_buzz(input)
+  begin
+    input = Float(input)
+  rescue ArgumentError => e
+    puts "#{input} is an improper value: #{e.class.name}, try again."
+    is_float = false
+  end
+
+  if is_float == false
+  else
+    if input%1 == 0
+    else
+      puts "Would you like to round your number #{input}? Y/N"
+      q = gets.chomp.downcase
+      if q == "y"
+        puts "Would you like to round your number #{input} UP? Y/N"
+        updown = gets.chomp.downcase
+        if updown == "y"
+          puts "Your number #{input} will be rounded UP."
+          input = input.ceil
+        else
+          puts "Your number #{input} will be rounded DOWN."
+          input = input.floor
+        end
+      end
+    end
+    print_fizz_buzz(input)
+  end
+end
+
+array_fb = []
+repeat = "y"
```

Comment:
setting variable - in this case `repeat` - before first `while` loop evaluation will work, but there's another loop that might accomplish it without setting variable before loop :)

Please research on `while` loop vs `do while` loop and look up how to write `do while` in Ruby :)

Reply:
Thanks for the suggestion - it sounds interesting. I'll educate myself on the topic and will come back with that part redesigned.